### ISSUE TITLE
qstat xml parsing can't handle empty joblist

### ISSUE THREAD

**Issue by gdevenyi** [pipitone/qbatch#82]
If qstat -x returns nothing, ET parsing fails, triggering an exception and breaking qbatch.

```
  File "<string>", line unknown
ParseError: no element found: line 1, column 0
```

qbatch stops at this point, but it should probably check for qstat -x having no output or at least warn that no dependencies are possible because of no jobs running.

**Comment by pipitone** [comment]
Good catch. Let's write a test for this and then a fix. :-)

I agree a warning here makes sense... perhaps only under verbose mode?

**Comment by gdevenyi** [comment]
Test is hard since Travis doesn't have a real qstat... 

Yes,  warn on verbose makes sense

**Comment by pipitone** [comment]
> Test is hard since Travis doesn't have a real qstat... 

Yeah, either factor out the call to qstat so that there is a testable function that only inspects the output, or mock the [call to subprocess](http://stackoverflow.com/questions/25692440/mocking-a-subprocess-call-in-python) or  [mock qstat](https://pythonhosted.org/MockProc/) (kinda cool actually).

Overkill?

**Comment by gdevenyi** [comment]
Overkill for now.

Fixed by #86
